src/include/stir/recon_buildblock/Parallelproj_projector/BackProjectorByBinParallelproj.h

```diff
@@ -75,9 +75,16 @@ class BackProjectorByBinParallelproj :
     This function has to be called before any back-projection is initiated.*/
   virtual void start_accumulating_in_new_target();
 
+  /// set defaults
+  void set_defaults();
+
   /// Set verbosity
   void set_verbosity(const bool verbosity) { _cuda_verbosity = verbosity; }
 
+   // set/get number of gpu chunks to use
+   void set_num_gpu_chunks(int num_gpu_chunks) {_num_gpu_chunks = num_gpu_chunks; }
+   int  get_num_gpu_chunks() { return _num_gpu_chunks; }
+
   BackProjectorByBinParallelproj* clone() const override;
```

Comment:
I had a merge conflict here. `BackProjectorByBinParallelproj* clone() const override;` does not exists on UCL/STIR but was on `tof_sino_UCL`

Reply:
sure. this is fine